.eslintrc.js

```diff
@@ -45,7 +45,7 @@ module.exports = {
   rules: {
     // default rules
     'no-console': 'off',
-    'no-unused-vars': 'warn', // change to 'error' 🙏
+    'no-unused-vars': ['warn', { "varsIgnorePattern": "^_" }], // change to 'error' 🙏
```

Comment:
Curious about this

Reply:
Yeah, allows you to do "_ignoreMe" to split up objects as I did to pull out the label. I know in `go` you just use a `_` variable for this.